src/NUnitEngine/nunit.common/nunit.common.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>NUnit</RootNamespace>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);618</NoWarn>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
+    <OutputPath>..\..\bin\$(Configuration)\</OutputPath>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)/../../Nullable.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\nunit.engine.api\nunit.engine.api.csproj" />
```

Comment:
If this library is (in the future) to be shared by other nunit components, then it shouldn't depend on 'engine.api'.
It then becomes where to define `InternalTraceLevel`. Maybe it should move to common instead.

Reply:
I agree it needs to go but I left it for now because `ITestEngine` has an `InternalTraceLevel` property. The problem will only arise if and when we want to create a separate package for the common, as you say. Can you think of an easy way to eliminate that property while still having the ability to set the engine trace level from the runner? Otherwise, let's leave it for a separate issue.

Reply:
If it is in the API, it should probably not be called InternalTraceLevel.
I only see an option splitting of a common.api but that might not be worth it if all itt does is hold an enum.
Probably leave it until another issue.